.zuul.yaml

```diff
@@ -2,6 +2,4 @@
     check:
```

Comment:
if you set:

  debug: True

we should get more info from zuul

Reply:
Added